### ISSUE TITLE
Argument list too long when linking versions

### ISSUE THREAD

**Issue by luizirber** [pirovc/genome_updater#16]
Hi!

I tried to update a previous run and got an error. Seems like `ln` didn't like linking 196k files. Maybe using `xargs` or `parallel` to link them individually?

Log:
```
$ genome_updater.sh -d "refseq" -g "archaea,bacteria,fungi,viral,protozoa" -c "all" -f "genomic.fna.gz" -o "refseq" -t 24 -u -m -a
----------------------------------------
      genome_updater version: 0.2.0
----------------------------------------
Mode: UPDATE - DOWNLOAD
Working directory: /data/databases/genome_updater/refseq/
----------------------------------------
Checking for missing files in the current version [2020-05-04_15-38-58]
 - None

Checking for extra files [2020-05-04_15-38-58]
 - None

Downloading assembly summary [2020-05-11_18-02-56]
 - 0/196639 entries removed [RefSeq category: all, Assembly level: all, Version status: latest]
 - 196639 entries available
Linking versions [2020-05-04_15-38-58 --> 2020-05-11_18-02-56]
bin/genome_updater.sh: line 692: ln: Argument list too long
```

**Comment by pirovc** [comment]
@luizirber thanks for spotting that. Could you give it a try with the fix and let me know if it works now?

Best
Vitor

**Comment by luizirber** [comment]
Yup, #17 works! Thanks!

**Comment by pirovc** [comment]
Nice, thanks. Fix merged with #19
